src/functionalTest/resources/application-functional.yaml

```diff
@@ -15,7 +15,7 @@ wa-post-deployment-test:
 azure:
   servicebus:
     connection-string: ${AZURE_SERVICE_BUS_CONNECTION_STRING}
-    topic: ${AZURE_SERVICE_BUS_TOPIC_NAME:ccd-case-events-aat}
+    topic: st-ccd-case-event
```

Comment:
I think this needs to be updated once the `AZURE_SERVICE_BUS_CONNECTION_STRING` is updated.
Currently calling https://portal.azure.com/#@HMCTS.NET/resource/subscriptions/1c4f0704-a29e-403d-b719-b90c34ef14c9/resourceGroups/ccd-shared-aat/providers/Microsoft.ServiceBus/namespaces/ccd-servicebus-aat/topics/ccd-case-events-aat/overview